docs/docs/3-features-modules/10-warning-module/index.md

```diff
@@ -0,0 +1,278 @@
+return {
+    module: AppModule,
+    imports: [
+        // other modules...
+        WarningsModule,
+    ],
+};
+```
+
+### 2. Add the Warnings Page to the Admin Menu
+
+To make the warnings accessible in the admin panel, add the following entry to the masterMenu:
+
+```typescript
+{
+    type: "route",
+    primary: <FormattedMessage id="menu.warnings" defaultMessage="Warnings" />,
+    route: {
+        path: "/system/warnings",
+        component: WarningsPage,
+    },
+    requiredPermission: "warnings",
+}
+```
+
+### 3. Schedule the Daily Warning Check
+
+To ensure warnings are checked regularly, schedule the following CLI command to run once per day:
+
+```
+npm run console check-warnings
+```
+
+It's recommended to run this job during off-peak hours (such as at night) to minimize impact on application performance.
+
+:::info
+
+The Warning Module comes with some built-in warnings out of the box.
+For example, if a SEO block is missing an HTML title, it will show a warning.
+
+:::
+
+## API: Custom Block Warnings
+
+You can define custom warnings for blocks by implementing a `warnings()` function in the API.  
+This function should return a list of warnings based on specific conditions.
+
+### Simple Block Warning
+
+Example:
+
+```typescript
+class NewsBlockData extends BlockData {
+    @BlockField({ nullable: true })
+    title?: string;
+
+    warnings(): BlockWarning[] {
+        const warnings: BlockWarning[] = [];
+
+        if (!this.title) {
+            warnings.push({ severity: "low", message: "missingTitle" });
+        }
+
+        return warnings;
+    }
+}
+```
+
+### Block Warning with Dependency Injection
+
+When you need dependency injection, you can also add your own WarningsService like that:
+
+#### Example of the Block
+
+```typescript
+class NewsBlockData extends BlockData {
+    @BlockField({ nullable: true })
+    id?: string;
+
+    warnings() {
+        return NewsBlockWarningsService;
+    }
+}
+```
+
+#### Example of the Service
+
+```typescript
+@Injectable()
+export class NewsBlockWarningsService implements BlockWarningsServiceInterface<NewsBlockData> {
+    constructor(
+        @InjectRepository(NewsRepository)
+        private readonly newsRepository: EntityRepository<NewsRepository>,
+    ) {}
+
+    async createWarnings(block: NewsBlockData): Promise<BlockWarning[]> {
+        const warnings: BlockWarning[] = [];
+
+        if (block.id) {
+            const news = await this.newsRepository.findOne({ id: block.id });
+            if (!news) {
+                warnings.push({ severity: "high", message: "invalidNewsTarget" });
+            }
+        } else {
+            warnings.push({ severity: "high", message: "missingNewsTarget" });
+        }
+        return warnings;
+    }
+}
+```
+
+## API: Custom Entity Warnings
+
+You can also create warnings for entities.  
+To do this, use the `@CreateWarning` decorator
+
+### Simple Entity Warning
+
+```typescript
+@CreateWarnings<News>(async (value) => {
```

Comment:
Is it CreateWarning or CreateWarnings? You're using both in the examples.

Reply:
CreateWarnings: 4ba496009096ad84c7558dbb1d9b889e2ba86292